pkg/ddc/jindo/transform.go

```diff
@@ -142,6 +142,7 @@ func (e *JindoEngine) transform(runtime *datav1alpha1.JindoRuntime) (value *Jind
 	e.transformTolerations(dataset, runtime, value)
 	e.transformResources(runtime, value)
 	e.transformLogConfig(runtime, value)
+	e.transformHostNetWork(runtime, value)
```

Comment:
I suggest the name is `transformNetworkMode`.

Reply:
ok, as the types define